sbncode/CAFMaker/FillReco.cxx

```diff
@@ -993,6 +993,19 @@ namespace caf
     }
   }
 
+  void FillCVNScores(const lcvn::Result *cvnResult,
+                           caf::SRSlice& slice)
+  { 
+
+    if ( cvnResult != nullptr ) {
+      auto const & cvn = cvnResult->fOutput;
+      slice.cvn.numuscore    = cvn[0][0];
+      slice.cvn.nuescore     = cvn[0][1];
+      slice.cvn.cosmicscore  = cvn[0][2];
+      slice.cvn.ncscore      = cvn[0][3];
```

Comment:
Is fOutput (std::vector<std::vector<float>>) guaranteed to have a second vector with 4 elements and primary vector with 1 element?
If not, this should probably be wrapped a try/catch.

Reply:
Thanks Dom. With the current network, it is guaranteed. But I added a check for the dimensions first.